python/ray/autoscaler/_private/gcp/config.py

```diff
@@ -47,6 +48,66 @@
 # with ServiceAccounts.
 
 
+def _validate_tpu_config(node: dict):
+    """Validate the provided node with TPU support.
+
+    If the config is malformed, users will run into an error but this function
+    will raise the error at config parsing time. This only tests very simple assertions.
+
+    Raises: `ValueError` in case the input is malformed.
+
+    """
+    if "acceleratorType" in node and "acceleratorConfig" in node:
+        raise ValueError(
+            "For TPU usage, acceleratorType and acceleratorConfig "
+            "cannot both be set."
+        )
+    if "acceleratorType" in node:
+        accelerator_type = node["acceleratorType"]
+        expected_pattern = re.compile(r"^v\d+[a-zA-Z]*-\d+$")
+        if not expected_pattern.match(accelerator_type):
+            raise ValueError(
+                "`acceleratorType` should match v(generation)-(cores/chips)"
```

Comment:
Will it print the `acceleratorType` that was actually received somewhere in the error traceback? If not, it would be helpful to print it here. Similarly for below

Reply:
Great point, added that functionality in